graph/src/data/query/error.rs

```diff
@@ -83,6 +83,63 @@ pub enum QueryExecutionError {
     ResultTooBig(usize, usize),
 }
 
+impl QueryExecutionError {
+    pub fn is_attestable(&self) -> bool {
+        use self::QueryExecutionError::*;
+        match self {
+            OperationNameRequired
+            | OperationNotFound(_)
+            | NotSupported(_)
+            | NoRootSubscriptionObjectType
+            | NonNullError(_, _)
+            | ListValueError(_, _)
```

Comment:
I _think_ a similar argument could apply to this error type, so out of caution I'd make it non-deterministic for now.